SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-intro.md

```diff
@@ -18,4 +18,4 @@ The exercise demonstrates that the workflows and iteration speed you’re used t
 
 
 <br/>------<br/>
-_2019-04-30 Page updated with limited editorial review_
\ No newline at end of file
+_2019-08-02 Page updated with limited editorial review_
```

Comment:
```suggestion
_2019-08-02 Page updated with limited editorial review: updated project name._</br>
_2019-04-30 Page updated with limited editorial review._
```

Reply:
doc out of date so will commit suggestions manually.